gofbink/gofbink.go

```diff
@@ -766,12 +766,9 @@ func (f *FBInk) PrintRBGA(xOff, yOff int16, im *image.RGBA, cfg *FBInkConfig) er
 
 // GetLastRect returns the last painted to area
 // See "fbink.h" for detailed usage and explanation
-func (f *FBInk) GetLastRect(rect *FBInkRect) {
+func (f *FBInk) GetLastRect() FBInkRect {
 	rectC := C.fbink_get_last_rect()
-	rect.Left = uint16(rectC.left)
-	rect.Top = uint16(rectC.top)
-	rect.Width = uint16(rectC.width)
-	rect.Height = uint16(rectC.height)
+	return FBInkRect{uint16(rectC.left), uint16(rectC.top), uint16(rectC.width), uint16(rectC.height)}
```

Comment:
I would probably explicitly label the fields during assignment, to avoid any potential struct order issues in the future.
```suggestion
	return FBInkRect{
	    Left: uint16(rectC.left), 
	    Top: uint16(rectC.top), 
	    Width: uint16(rectC.width), 
	    Height: uint16(rectC.height),
	}
```

Reply:
Cool, was hoping something like this was possible, but wasn't sure of the proper syntax ;).

Reply:
Yep, pretty straightforward.

BTW, if you ever want to test a bit of syntax or functionality, the [Go Playground](https://play.golang.org/) provides an environment to play in.

And if you ever want to get further aquainted with the language, [A tour of Go](https://tour.golang.org/welcome/1) is a great place to quickly grasp the basics.